### PR TITLE
Update Breadcrumb Styles

### PR DESCRIPTION
**Pull request by Blendify**
- Fixes extra padding to the left/right
- Used slash instead of >>
- Make literalls match other link style
- Adds demo docs for testing

Before:

Screenshot from 2021-03-11 14-48-45

After:

image

### CODE REVIEW

File: src/sass/_theme_breadcrumbs.sass [hunk 1]
```diff
@@ -1,21 +1,19 @@
...
-    padding: 5px
-    border: none
-    background: none
-    &.literal
-      color: $text-color
+    all: inherit
+    color: inherit
+
+.breadcrumb-item::before
+  content: "/ "
```
Comment (nienn) [inline src/sass/_theme_breadcrumbs.sass:15]: The look of the new breadcrumb separators resulted a bit too intrusive.
I would maybe shrink them and make them lighter.

```suggestion
  content: "/"
  color: #bbb
  font-size: 13px
  padding: 0 6px 0 3px
```
_The new padding is required to re-center them, after changing the font size._

**Comment by nienn** [comment]
Changes look good!
I don't particular like the new separator style — made a suggestion on that — but this might be more of a personal preference.

Now that we can test a long breadcrumbs path directly on the docs I'm also noticing that the floating right `.wy-breadcrumbs-aside` does not sit well on small screens.
**Before:**
b-before


**After:**
b-after

**Comment by benjaoming** [comment]
I really like the PR and @nienn's improvement! I did a quick verification and ended up with exactly the same results :+1: 

Before @nienn's suggestion:

image

After @nienn's suggestion:

image

File: docs/demo/level1/level2/index.rst [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
+******************************************
+:mod:`Breadcrumb Level 2 <test_py_module>`
```
Comment (benjaoming) [inline docs/demo/level1/level2/index.rst:3]: Very clever test this!

**Review by benjaoming — APPROVED** [review]
This is a really positive improvement and adds a very nice demo with 6 nested sections :+1: 

I don't see any risks of regressions here.

File: src/sass/_theme_breadcrumbs.sass (hunk 1 above)
Comment (olivaja) [reply src/sass/_theme_breadcrumbs.sass]: Hola cómo estás

**Pull request merged**